stylesheets/modules/_footer.scss

```diff
@@ -9,9 +9,15 @@
 
   .cite {
     @include clearfix();
+    display: grid;
+    grid-template-columns: 1fr;
     font-size: 1.25em;
     margin-bottom: $gutter-width;
-    max-width: 22em;
+
+    @media screen and (min-width : $medium-screen) {
+      grid-template-columns: auto 1fr;
+      max-width: 22em;
```

Comment:
I notice use of 22em in the `_header` and `_footer` modules. Consider creating a Sass variable to make this code less brittle.

Reply:
Good point. And it gave me another chance to look at these lines.

Now that I look at them again, it seems `max-width` can be replaced by a size range of `grid-template-columns` since the elements are grid items. And that ends up with different values of size for each grid layout which demotivates me to create variables for.